app/templates/base.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" color-mode="regular">
```

Comment:
Don't use made-up attribute names :)
If you want to, you can use `data-attributes` to do it

Reply:
not sure i understand. i followed some articles about this and thats the way they described. do you mean adding "data" before? "data-color-mode"?

Reply:
Yeah. You shouldn't add attributes that aren't in the standard. There are no standardized `color-mode`  attribute.

![image](https://user-images.githubusercontent.com/1492939/109171841-d2c12e80-778a-11eb-94e9-8cbb130d5813.png)